common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -238,8 +267,63 @@ export class SendTransaction extends React.Component {
     return { to, data, value, unit, gasLimit, readOnly };
   }
 
+  isValid() {
+    const { to, value } = this.state;
+    return (
+      isValidETHAddress(to) &&
+      value &&
+      Number(value) > 0 &&
+      !isNaN(Number(value)) &&
+      isFinite(Number(value))
+    );
+  }
+
+  // FIXME MOVE ME
+  getTransactionFromState() {
+    // FIXME add gas price
```

Comment:
Most field are optional when estimating, we still want to send gasprice because there are ico contracts that check it and it might or might not change estimate